adminguide.rst

```diff
@@ -770,4 +816,3 @@ circular buffer.  For some problems, it may be useful to view this log:
 ~~~~~~~~~~~~~
 KVS Eventlogs
 ~~~~~~~~~~~~~
-
```

Comment:
Looks like a stray whitespace change here (didn't track down which commit it came from though, sorry)

Reply:
🤦‍♂️ I'm sorry about that; I must have left that over when I moved the accounting section. Just pushed a fix to remove that whitespace change, so that should be fixed now.